### PR TITLE
GenericHybrid with DimensionalData and extra_loss function

### PR DESCRIPTION
**Pull request by BernhardAhrens**
extra_loss function can be defined like so. Here steering rb to 5

```
extra_loss = function (ŷ)
    (; a=sum((5.0 .- ŷ.rb) .^ 2))
end
```

### CODE REVIEW

File: src/utils/logging_loss.jl [hunk 1]
```diff
@@ -172,7 +149,7 @@ function compute_loss(ŷ, y, y_nan, targets, loss_spec, agg::Function)
 end
 
 function compute_loss(ŷ, y::AbstractDimArray, y_nan::AbstractDimArray, targets, loss_spec, agg::Function)
-    losses = [_apply_loss(ŷ[k], y[col = At(k)], y_nan[col = At(k)], loss_spec) for k in targets]
+    losses = [_apply_loss(ŷ[k], y[col=At(k)], y_nan[col=At(k)], loss_spec) for k in targets]
```
[the remark this answers is not in the document]
Comment (BernhardAhrens) [reply src/utils/logging_loss.jl]: Maybe we just rename whatever the user gives and print something original axis x renamed to row, other axis untouched
